src/main/java/abaz/calculator1/controller/CalculatorController.java

```diff
@@ -0,0 +1,90 @@
+package abaz.calculator1.controller;
+
+import abaz.calculator1.settings.CalculatorSettings;
+import abaz.calculator1.settings.CalculatorSettingsImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/calculator1")
+public class CalculatorController {
+    private final CalculatorSettings calculatorSettings;
+
+    public CalculatorController(CalculatorSettings calculatorSettings) {
+        this.calculatorSettings = calculatorSettings;
+    }
+
+    @GetMapping("/info")
+    public String ShowAnimation(){
+        test();
+        return "Добро пожаловать в калькулятор";
+    }
+    void test(){
+        int sum = calculatorSettings.sum(2, 4);
+        System.out.println(sum);
+    }
+    @GetMapping("/plus")
+    public String sumNumbers(@RequestParam(required = false) Integer num1, @RequestParam(required = false) Integer num2){
+        if(num1 == null || num2 == null){
+            return "Неверные данные";
+        }
+        int result = calculatorSettings.sum(num1, num2);
+        return generateMessage(num1, num2, '+' , result);
+    }
+    private String generateMessage(int num1, int num2, char action, int result){
+        return String.format("%d %c %d = %d", num1, '+' , num2, result);
```

Comment:
```suggestion
        return String.format("%d %c %d = %d", num1, action , num2, result);
```
здесь нужно передать операцию, а то у тебя будет для всех операций +